package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "werkbot-framewerk",
-  "version": "2.1.0",
+  "version": "2.1.2",
```

Comment:
I think this should be 1.2.1 https://github.com/werkbot/framewerk/releases

Reply:
Shouldn't that stay "2.1.2"? Since we want master have all the latest changes?

I suppose maybe we need a 1 branch for the 1.x releases....

Maybe I will hold off until we get our branching model in order

Reply:
My mistake. I meant this should be the latest release tag 2.1.1

Reply:
Only issue is, why is this 2.1.2 on the 2.11 branch? It should be 2.1.1 on that branch as well

Reply:
This commit was picked up as a patch, so there was likely a draft release generated for 2.1.2 at some point as well https://github.com/werkbot/framewerk/commit/744e18d64a6e32cdf0e87984d38c90d17b3735e9